jobs.html

```diff
@@ -0,0 +1,125 @@
+
+
+                            
+                            <a href="/Events" class="navbar-item">
+                                Events
+                            </a>
+                            
+
+                            
+                            <a href="/Food" class="navbar-item">
+                                Food
+                            </a>
+                            
+                            
+                            <!-- this means a User can only view this page if they are logged out, This page will be hidden when user is logged in  -->
+                            
+
+                            
+                            <a href="/AboutUs" class="navbar-item">
+                                About Us
+                            </a>
+                            
+
+                            <!-- this means a User can only view this page if they are logged in, This page will be hidden when user is logged out  -->
+                            
+                            <a href="/Profile" class="navbar-item">
+                                Profile
+                            </a>
+                            
+
+                            <!-- this means a User can only view this page if they are logged in, This page will be hidden when user is logged out  -->
+                            
+                            <a href="/Logout" class="navbar-item">
+                                Logout
+                            </a>
+                            
+                            
+                            <!-- Todo: Add and Route other HTML UI pages (e.g: Welcome Page) -->
+                            
+
+                        </div>
+                    </div>
+                </div>
+            </nav>
+        </div>
+
+
+        <div class="hero-body">
+            <div class="container has-text-centered">
+                
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="Attractions.css"><!--link to styles sheet-->
+<link rel="stylesheet" href="/static/css/Attractions.css">
+<link rel="stylesheet" href="/static/css/index.css">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8">
+    <title>Jobs</title> 
+   <body>
+    <h1 class="title">
+        Jobs Page
+      </h1>
+      
+      
+</body>
+
+</html>
+<div class="card" style="width: 18rem;">
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">Cras justo odio</li>
+      <li class="list-group-item">Dapibus ac facilisis in</li>
+      <li class="list-group-item">Vestibulum at eros</li>
+    </ul>
+  </div>
+  <div class="card" style="width: 18rem;">
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">Cras justo odio</li>
+      <li class="list-group-item">Dapibus ac facilisis in</li>
+      <li class="list-group-item">Vestibulum at eros</li>
+    </ul>
+  </div>
```

Comment:
Code looks good, but is this chunk of code here twice on accident or is it a placeholder?